integration/package.json

```diff
@@ -4,39 +4,41 @@
   "version": "0.6.0",
   "license": "MIT",
   "scripts": {
-    "test": "cross-env NODE_ENV=test jest",
-    "test:nocache": "cross-env NODE_ENV=test jest --no-cache",
+    "cypressJobServer": "node cypress/support/cypressJobServer.js",
```

Comment:
How do you feel about making this Typescript and running with `tsc ypress/support/cypressJobServer.ts`?

Also we should support windows by running it with `cross-env` e.g. `cross-env tsc ypress/support/cypressJobServer.ts`